driver-dse/src/main/java/com/datastax/driver/graph/Edge.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.graph;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
+ * A default representation of an Edge in DSE Graph.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonDeserialize(using = DefaultEdgeDeserializer.class)
+public class Edge {
```

Comment:
Since edge and vertex share some characteristics (id, label, type), should we have a parent class like gremlin has with [Element](http://tinkerpop.incubator.apache.org/javadocs/3.1.1-SNAPSHOT/full/org/apache/tinkerpop/gremlin/structure/Element.html)?  The other thing I wonder is are we going to end up duplicating a lot of the gremlin java API, should we just use the gremlin API proper, or is that bringing in too much (Would add gremlin as a dependency)?  (apologies if this has already been discussed).